minefana-bungeecord/src/main/java/com/github/games647/minefana/collectors/BungeePlayerCollector.java

```diff
@@ -88,7 +88,15 @@ public void onServerSwitch(ServerSwitchEvent switchEvent) {
     }
 
     private String getCurrentServer(ProxiedPlayer player) {
-        return player.getServer().getInfo().getName();
+        String name;
+        try {
+            name = player.getServer().getInfo().getName();
+
+        }catch (Exception e){
+            System.out.println("Error getting server name");
```

Comment:
Please use the logger that including posting the stacktrace. Otherwise it would make it harder to trace a bug. Ignoring stacktraces isn't really a good practice in most cases.